bitcoin/src/blockdata/constants.rs

```diff
@@ -46,8 +46,6 @@ pub const MAX_WITNESS_SCRIPT_SIZE: usize = 10_000;
 pub const MAX_STACK_ELEMENT_SIZE: usize = 520;
 /// How may blocks between halvings.
 pub const SUBSIDY_HALVING_INTERVAL: u32 = 210_000;
-/// Maximum allowed value for an integer in Script.
-pub const MAX_SCRIPTNUM_VALUE: u32 = 0x80000000; // 2^31
```

Comment:
The commit message/PR says deprecate but this removes it. Also we really should deprecate it and not remove it. Suggested deprecation message: "the constant has ambiguous semantic, define your own"

Reply:
Sorry, I misunderstood the meaning of "deprecate".
I added deprecated attribute with the message.